README.md

```diff
@@ -3959,3 +3959,13 @@ My name is Vlad!
 
 ## Hello, kottans!
 Hello, kottans!
+
+# kottans-frontend
+**Hello** <br>
+
+<sub> My name is Kate :wink:</sub> <br> Thanks for your course. <br> Git is new for me but I begin to understand how it works, thanks to you 😃 <br>A lot of new information/ The most intresting is 
```

Comment:
This looks to me like an HTML attribute <br> ... 
Let's be consistent and use only markdown pls https://www.markdownguide.org/basic-syntax/